app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::UsersController < ApplicationController
+  def show
+    render json: UserSerializer.new(User.find(params[:id]))
```

Comment:
I think this should have some error handling such as a rescue block when the user is not found, which is a possibility.